### DISCUSSION TITLE
Can't change the target branch of an existing PR

### DISCUSSION THREAD

**Discussion by bertyhell** [community/community#60328] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

When i try to update the target branch of an exisitng PR, the dropdown to select the branch does not appear

steps:
* go to a PR
* click edit
* click on the "main" branch dropdown

expected behavior: a dropdown is shown to select a different target branch

current behavior: the dropdown is not shown, the cursor changes though.

![image](https://github.com/community/community/assets/1710840/ec128a0d-22f9-4407-b0fd-25fb011d015d)

video:
https://github.com/community/community/assets/1710840/21bb0aec-26bc-4aed-af81-1ad7de652753

**Comment by brcarp** [comment]
I created a support ticket in the portal through my organization's account. I'll post an update if I get any response.

**Reply by brcarp** [reply, marked as answer]
GitHub is indicating that it was resolved: https://www.githubstatus.com/incidents/hz717kx2gsvq

**Reply by bertyhell** [reply]
confirmed, its working now

thx
